src/main/resources/swagger/api-docs.yaml

```diff
@@ -501,6 +501,11 @@ definitions:
           The archive must be in tar.gz format, must not include a parent directory, and
           must have an entry point named 'main'. For more information on notebook extensions,
           see http://jupyter-notebook.readthedocs.io/en/latest/extending/frontend_extensions.html.
+      jupyterUserScriptUri:
+        type: string
+        description: |
+          Optional bucket URI to a bash script the user wishes to run inside their jupyter docker.
+          The script must be in .sh format and must not include a parent directory.
```

Comment:
We can probably just delete this second sentence? I'm not sure it needs to be in .sh format: I could name it `my_awesome_script` and we would `chmod +x` it and try to execute it. And "must not include a parent directory" doesn't make sense for scripts. :)